builder/entrypoint.sh

```diff
@@ -70,7 +70,7 @@ run_tracee_rules() {
     events=$($TRACEE_RULES_EXE --list-events)
 
     echo "INFO: starting tracee-ebpf..."
-    ${TRACEE_EBPF_EXE} --metrics --output=format:gob --output=option:parse-arguments --trace event=${events} --output=out-file:${TRACEE_PIPE} &
+    ${TRACEE_EBPF_EXE} --metrics --output=format:gob --output option:protocol --output=option:parse-arguments --trace event=${events} --output=out-file:${TRACEE_PIPE} &
```

Comment:
Do we need this new option after all? What would be a use case to not passing the `--ouput=option:protocol` flag to tracee-ebpf? I though that tracee-ebpf is going to send generic event and tracee-rules is going to consume generic event.

```suggestion
    ${TRACEE_EBPF_EXE} --metrics --output=format:gob --output=option:protocol --output=option:parse-arguments --trace event=${events} --output=out-file:${TRACEE_PIPE} &
```

Reply:
I thought that it would be the outputting client's responsibility to send a protocolized event, not the producer itself (in this case i'm making the distinction between the cmd/tracee-ebpf program and the Tracee struct itself). 

I would like the protocol toggle so I could for example dump the events into a file, without all the Headers metadata around it (doubly so if and when we add more headers). Maybe it would be more appropriate to send as protocol by default and disable it if wanted, but I think it would make even less sense since it would mean by default Table and JSON/GOB formats have different behaviours (table would in any case output the raw event).

Personally, I think the best solution would have been to separate formatting and type output decision, but there's no clean way to do it without generics (not yet in Go) or `interface{}` which would be inefficient.

Reply:
Resolving since the option was removed.